src/common/constants.js

```diff
@@ -3981,12 +3981,293 @@ const yesNo = [
 
 const usaStates = _.map(states.USA, (stateData) => { return stateData.value });
 
```

Comment:
FYI, I stole these countries and states by inspecting the `countries` and `states` responses from Letters Confirm Address